tests/unit/cloud/clouds/test_xen.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
Since this is a new test can you add it as a pytest in tests/pytests/unit/cloud/clouds? We are trying to migrate all our tests to pytest.

Reply:
bump ^, I can also do the work here as well just let me know.

Reply:
@Ch3LL I am not sure what needs to happen for this to use purest. Do you have some docs about the change?

Reply:
We have tried to update most of our testing docs to mirror the use of pytest but I do not think we have specific docs related to migrating tests from our previous test runner to pytest. I can do it this time. I'll try to find time tomorrow.

Reply:
I was actually able to do it right now and just pushed the test. I only need to do a couple things:
1. remove the class
2. use `configure_loader_modules` instead of `setup_loader_modules`
3. update the assertions to normal python asserts.